### PR TITLE
cut: fix a command example & minor README.md improvements

### PR DESCRIPTION
**Pull request by PetrusKiendys**
----

- [ ] The page (if new), does not already exist in the repo.

- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

- Fixup command example for **cut**
- Minor changes in README.md

### CODE REVIEW

File: scripts/pdf/README.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ This directory contains the script and related resources to generate a PDF copy
 
 ## Requirements
 
-The PDF is generated by first converting the markdown files to HTML, and then rendering those HTML files as PDF. It depends on the following libraries-
+The PDF is generated by first converting the markdown files to HTML, and then rendering those HTML files as PDF. It depends on the following libraries:
 
-#### Python-Markdown
+**Python-Markdown**
```
Comment (agnivade) [inline scripts/pdf/README.md]: Let us revert these. `####` is a correct markdown syntax and is consistent with `##` above.

Comment (PetrusKiendys) [reply scripts/pdf/README.md]: Alright sure, I have allowed edits to be done from maintainers.
Will you edit the PR or should I do it?

File: pages/common/cut.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 - Cut out everything from the 3rd character to the end of each line:
 
-`cut -c{{3-}}`
+`cut -c {{3-}}`
```
Comment (sbrl) [inline pages/common/cut.md]: This was valid syntax before. The space here is completely optional actually. Personally, I'd like to leave this as-is - as some commands (e.g. 7-zip) require the lack of space, and that can be confusing if it's a one-off thing.

Comment (agnivade) [reply pages/common/cut.md]: I think it is more to do with consistency. If you see the above 2 examples, they also use `-c` but have a space. I do not mind either way, but if we are not using space, then I would change the above examples too.

Comment (sbrl) [reply pages/common/cut.md]: Ah. Let's change this then. I've just seen a number of PRs making this kind of change recently.

Comment (PetrusKiendys) [reply pages/common/cut.md]: Yes, exactly as @agnivade puts it, this change is for the sake of consistency. Either we omit the space or insert it, either one of those but preferably the same in all examples.

Sorry for replying late to this, luckily @agnivade motivated the change for me, thanks!

**Pull request merged**